.gitignore

```diff
@@ -14,8 +14,6 @@ webpack-stats.json
 htmlcov
 .tox
 .pytest_cache
```

Comment:
Why are these yarn files these removed from the `.gitignore`? We probably wouldn't want to track them.

Reply:
Oh that's my bad, I'll swap it back